tests/toDo.test.js

```diff
@@ -0,0 +1,72 @@
+import ToDo from "../src/models/toDo";
+import ToDoList from "../src/models/toDoList";
+import { describe, it, expect } from "vitest";
+
+describe("ToDo", () => {
+  it("should create a new ToDo", () => {
+    const form = document.createElement("form");
+    const input = document.createElement("input");
+    input.name = "description";
+    input.value = "Buy milk";
+    form.appendChild(input);
+
+    const toDo = new ToDo(form);
```

Comment:
@sal-wtag vai, I can see we are repeating this portion of code throughout the describe block.
So, I would suggest to handle this using a dedicated function

```suggestion
  function createMockForm / createMockTodo(inputValue) {
    const form = document.createElement("form");
    const input = document.createElement("input");
    
    input.name = "description";
    input.value = value;
    
    form.appendChild(inputValue);
    
    return form;
  }
  
  beforeEach(() => {
    const mockForm = createMockForm("Buy milk");
  });
  
  it("should create a new ToDo with valid properties", () => {
    const toDo = new ToDo(mockForm);
```